### DISCUSSION TITLE
[잡생각] 일머리 문해력이라는 책에 나오는 내용입니다

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#296] [General]
![image](https://user-images.githubusercontent.com/18538418/230824744-708fcbee-7da8-49fc-b41b-d763b94bf920.jpeg)

하루마다 급변하는 세상에서 우리는 어떻게 대응해야하는지에 대한 저자의 답 입니다

저희 상황에 맞춰 생각해보면,
접하고 ~ 배우고 까진 책모임 때 진행하고있으니, 그 이후인 극복하고, 적응하는 실천의 영역에 좀 더 신경쓰면되지않을까? 라는 잡생각이들었습니다 ㅎㅎ

**Comment by fkdl0048** [comment]
좋은 글귀 감사합니다..! 이런 글을 접할 때 마다 "그래도 잘 걷고 있구나"라는 생각이 들어서 기분이 좋습니다..!

**Comment by jongfeel** [comment]
워크샵때 academic conference에서 읽었던 좋은 책의 실천 사례가 있다면 얘기해 보는 시간을 가지는 것도 좋을 것 같다는 생각이 드네요.
